app.js

```diff
@@ -0,0 +1,450 @@
+    },
+    
+    createEvents() {
+        const memberNameInput = document.querySelector('.member-name-input');
+        memberNameInput.addEventListener('change', handlers.checkGetMemberName.bind(this));
+    },
+
+    getNextWeekend(day='') {
+        const todaysDate = day || new Date();
+        const todaysIndex = todaysDate.getDay();
+        
+        if (todaysIndex === 6) { // func 
+            this.storeWeekend(todaysDate);
+        } else {
+            const nextDay = todaysDate.setDate(todaysDate.getDate() + 1);
+            const day = new Date(nextDay);
+            this.getNextWeekend(day);
+        }
+    },
+    
+    storeWeekend(todaysDate) {
+        const weekend = [];
+        const saturday = todaysDate.toLocaleDateString();
+        const sunday = new Date(todaysDate.setDate(todaysDate.getDate() + 1)).toLocaleDateString();
+        weekend.push(saturday, sunday);
+        document.querySelector('.saturday-display').textContent = weekend[0];
+        document.querySelector('.sunday-display').textContent = weekend[1];
+        return weekend;    
+    },
+    
+    createProfileContainer(className) {
+        const article = document.createElement('article');
+        article.classList.add(className);
+        return article;
+    },
+    
+    createProfileHeading(person, className) {
+        const heading = document.createElement('h2');
+        heading.textContent = person.name;
+        heading.classList.add(className);
+        return heading;
+    },
+    
+    createProfileStartDate(person, className) {
+        const label = document.createElement('label');
+        label.textContent = person.date;
+        label.classList.add(className);
+        return label;
+    },
+    
+    createProfileGroup(person, className) {
+        const label = document.createElement('label');
+        label.textContent = person.group;
+        label.classList.add(className);
+        return label;
+    },
+    
+    createProfileRecommendation(person, className) {
+        const label = document.createElement('label');
+        label.textContent = person.recommendedBy;
+        label.classList.add(className);
+        return label;
+    },
+        
+    createProfile() {
+        const list = app.listOfPeople;
+        list.forEach(person => {
+            const article = this.createProfileContainer('person-shower');
+            const heading = this.createProfileHeading(person, 'name-shower');
+            const startDate = this.createProfileStartDate(person, 'start-date-shower');
+            const group = this.createProfileGroup(person, 'group-shower');
+            const recommendation = this.createProfileRecommendation(person, 'recommendation-shower');
+            article.appendChild(heading);
+            article.appendChild(startDate);
+            article.appendChild(group);
+            article.appendChild(recommendation);
+            const profile = document.querySelector('.profile');
```

Comment:
we only need to declare `profile` only once so let's move it outside `.forEach()`